### DISCUSSION TITLE
Quasar Keycloak Redirection Issue

### DISCUSSION THREAD

**Discussion by rkrajukhunt** [quasarframework/quasar#13826] [CLI - Capacitor / Cordova modes]
I am using Vue 3 with Vita and keycloak 

Keycloak redirect is working fine in-app, when I am trying to build an app for mobile using capacitor build, trying to check-in ios,  keycloak are not redirected into app, shows invalid redirect URI

can anyone help to set up the redirect URL for keycloak to mobile app as well as set up a universal link ios

**Comment by cdecinkoKnight** [comment]
Can you please share your code? I am setting up Keycloak with Quasar for the first time.
